### PR TITLE
Update Zeek to 4.0.5

### PR DESCRIPTION
**Pull request by JonZeolla**
# Summary of the contribution
Update Zeek to 4.0.5 and librdkafka to 1.4.4.  Updating librdkafka to the latest release failed; separating that effort for another time.

## Breaking change
This removes support for Centos 8, which was already non-functional.

## Testing
Leverage built-in tests.

## Checklist
- [X] Confirm that any associated issues are indicated in the pull request title
- [X] Rebase your PR against the latest commit within the target branch
- [X] Include steps to verify and test the intended change
- [X] Write or update unit tests and/or integration tests to verify your changes
- [X] Run shellcheck against any new or updated shell scripts, indicating the reasoning behind any disabled checks
- [X] Indicate whether or not this is a breaking change
- [X] Ensure that any new dependencies are these dependencies licensed in a way that is compatible for inclusion under [ASF 2.0](http://www.apache.org/legal/resolved.html#category-a)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -52,7 +52,7 @@ A Zeek log writer that sends logging output to Kafka, providing a convenient mea
     Verify the following REQUIRED external dependencies:
     (Ensure their installation on all relevant systems before proceeding):
       from zeek/seisollc/zeek-kafka (1.0.0):
-        librdkafka ~1.4.2
+        librdkafka ~1.4.4
```
Comment (JonZeolla) [inline README.md:55]:
```suggestion
        librdkafka ~1.4.2
```

File: zkg.meta [hunk 2]
```diff
@@ -10,6 +10,6 @@ depends =
   zeek >=4.0.0
   zkg >=2.0
 external_depends =
```
Comment (ottobackwards) [inline zkg.meta:12]: I don't understand why it is 1.4.4 other places and 1.4.2 here?

Comment (JonZeolla) [reply zkg.meta:12]: It's `~1.4.2` meaning we depend on this version or anything greater in the same minor range (i.e. 1.4.x).  We support as old as 1.4.2, or newer, but our tests run against 1.4.4.  That's why I made sure it was still 1.4.2 in 20381521089051ab7ee4dd0ac2ad887228587b79

Comment (ottobackwards) [reply zkg.meta:12]: but we want it less than the latest?

Comment (JonZeolla) [reply zkg.meta:12]: @ottobackwards the latest release failed our tests out of the box so I reverted to this patch update for the time being

**Review by ottobackwards — APPROVED** [review]

**Pull request merged**